README.md

```diff
@@ -77,15 +63,34 @@ python3 -m topic_benchmark run -e intfloat/e5-large-v2
 
 The results for each embedding model will be found in the `results` folder (unless a value for `--out_file` is explicitly passed).
 
-Each of these commands will compute results for all metrics, all datasets, all topic modeling methods, and all hyperparameters (i.e., number of topics) reported in the paper given a specific embedding model. Note that running these commends can be therefore be very time consuming, due to the large number of models that are estimated and evaluated.
+To produce figures and tables in the paper, you can use the scripts in the  `s3_paper_scripts/` folder.
+```bash
+pip install -r s3_paper_scripts/requirements.txt
+
+# Table 3: Main Table (tables/main_table.tex)
+python3 s3_paper_scripts/effect_of_preprocessing.py
```

Comment:
this should be `python3 s3_paper_scripts/main_table.py`, right?

Reply:
Yes